### ISSUE TITLE
blocked connection

### ISSUE THREAD

**Issue by odi-um** [antoniomika/sish#34]
hi @antoniomika,

not sure if its directly related to the software you've created but probably something obvious and I just can't figure this out.

I've configured sish on DO droplet (centos 7) using docker. Problem that I'm experiencing is a little bit weird. 

Opened tunnel from my local machine port 6767 to remote server with sish. When I try to check the page in the browser, I'm seeing loading state of the tab for eternity. When I close tunnel, content of the page appears immediately. 

Test page is a simple PHP script in one line which does echo and that's it, its working as it should when accessed directly.

It looks like browser sits and waits when connection will be closed by server. Same thing appears when I try to debug http connection with httpstat.

debug output of ssh client started with -vvv looks like this 

```
Starting SSH Fowarding service for http:80. Forwarded connections can be accessed via the following methods:
HTTP: http://xxx:80
HTTPS: https://xxx:443
debug3: receive packet: type 99
debug2: channel_input_status_confirm: type 99 id 0
debug2: PTY allocation request accepted on channel 0
debug3: receive packet: type 90
debug1: client_input_channel_open: ctype forwarded-tcpip rchan 1 win 2097152 max 32768
debug1: client_request_forwarded_tcpip: listen localhost port 80, originator localhost port 80
debug2: fd 9 setting O_NONBLOCK
debug2: fd 9 setting TCP_NODELAY
debug1: connect_next: host 127.0.0.1 ([127.0.0.1]:6767) in progress, fd=9
debug3: fd 9 is O_NONBLOCK
debug3: fd 9 is O_NONBLOCK
debug1: channel 1: new [localhost]
debug1: confirm forwarded-tcpip
debug3: channel 1: waiting for connection
debug1: channel 1: connected to 127.0.0.1 port 6767
debug3: send packet: type 91
debug2: channel 1: read<=0 rfd 9 len 0
debug2: channel 1: read failed
debug2: channel 1: chan_shutdown_read (i0 o0 sock 9 wfd 9 efd -1 [closed])
debug2: channel 1: input open -> drain
debug2: channel 1: ibuf empty
debug2: channel 1: send eof
debug3: send packet: type 96
debug2: channel 1: input drain -> closed
debug2: channel 1: rcvd adjust 175
```

When I try to setup ssh tunnel without using sish (on same server), everything works like a charm so its definitely not a server misconfiguration. 

Thank you for your help in advance.

**Comment by antoniomika** [comment]
Hmm, I haven't seen that before! How's the php server setup? I'm trying to get more information so I can potentially try to reproduce this.

**Comment by antoniomika** [comment]
Hi @odi-um!

I was able to reproduce the issue have a possible fix. Would you be interested in testing it? You can test it with the docker image tagged `6d41a7f6a8b2e90fa782ec3327978fb76f7e10ad` (`antoniomika/sish:6d41a7f6a8b2e90fa782ec3327978fb76f7e10ad`) if you use docker. Otherwise the fix is at #39

**Comment by odi-um** [comment]
@antoniomika tested and working, thank you.
